internal/collectors/uuid.go

```diff
@@ -18,9 +18,15 @@ type UUID struct{}
 func (UUID) run(arch string) (Result, error) {
 	var uuid string
 	var err error
+
 	switch arch {
+	case ARCHITECTURE_ARM64:
+		uuid, err = uuidDefault()
+		if err == nil && uuid == "" {
```

Comment:
Can you add a comment here why this is happening. Aka describing the non ACPI platforms do not provide a `uuid` via `dmidecode`